meerk40t/gui/panes/navigationpanels.py

```diff
@@ -933,104 +1029,148 @@ def matrix_updated(self):
         dy = 0
-        self.context("translate %f %f\n" % (dx, dy))
-        self.matrix_updated()
+        self.translateit(dx, dy)
+
+    def on_translate_left_10(self, event=None):  # wxGlade: Navigation.<event_handler>
+        dx = -self.context.navigate_jog * 10
+        dy = 0
+        self.translateit(dx, dy)
 
-    def on_translate_right(self, event=None):  # wxGlade: Navigation.<event_handler>
+    def on_translate_right_1(self, event=None):  # wxGlade: Navigation.<event_handler>
         dx = self.context.navigate_jog
         dy = 0
-        self.context("translate %f %f\n" % (dx, dy))
-        self.matrix_updated()
+        self.translateit(dx, dy)
+
+    def on_translate_right_10(self, event=None):  # wxGlade: Navigation.<event_handler>
+        dx = self.context.navigate_jog * 10
+        dy = 0
+        self.translateit(dx, dy)
 
-    def on_translate_down(self, event=None):  # wxGlade: Navigation.<event_handler>
+    def on_translate_down_1(self, event=None):  # wxGlade: Navigation.<event_handler>
         dx = 0
         dy = self.context.navigate_jog
-        self.context("translate %f %f\n" % (dx, dy))
-        self.matrix_updated()
+        self.translateit(dx, dy)
+
+    def on_translate_down_10(self, event=None):  # wxGlade: Navigation.<event_handler>
+        dx = 0
+        dy = self.context.navigate_jog * 10
+        self.translateit(dx, dy)
 
     def on_reset(self, event=None):  # wxGlade: Navigation.<event_handler>
         self.context("reset\n")
         self.matrix_updated()
 
-    def on_rotate_ccw(self, event=None):  # wxGlade: Navigation.<event_handler>
-        spooler, input_driver, output = self.context.registered[
-            "device/%s" % self.context.root.active
-        ]
-        x = input_driver.current_x if input_driver is not None else 0
-        y = input_driver.current_y if input_driver is not None else 0
-        self.context("rotate %fdeg %f %f\n" % (-5, x, y))
-        self.matrix_updated()
+    def on_rotate_ccw_5(self, event=None):  # wxGlade: Navigation.<event_handler>
+        angle = -5.0
+        self.rotateit(angle)
 
-    def on_rotate_cw(self, event=None):  # wxGlade: Navigation.<event_handler>
-        spooler, input_driver, output = self.context.registered[
-            "device/%s" % self.context.root.active
-        ]
-        x = input_driver.current_x if input_driver is not None else 0
-        y = input_driver.current_y if input_driver is not None else 0
-        self.context("rotate %fdeg %f %f\n" % (5, x, y))
-        self.matrix_updated()
+    def on_rotate_cw_5(self, event=None):  # wxGlade: Navigation.<event_handler>
+        angle = 5.0
+        self.rotateit(angle)
+
+    def on_rotate_ccw_90(self, event=None):  # wxGlade: Navigation.<event_handler>
+        angle = -90.0
+        self.rotateit(angle)
+
+    def on_rotate_cw_90(self, event=None):  # wxGlade: Navigation.<event_handler>
+        angle = 90.0
+        self.rotateit(angle)
+
+    def skewed_value(self, stxt):
```

Comment:
Because `self` is not used inside this method, codacy is recommending that you add the `@staticmethod` decorator immediately before and remove `self` from the parameters.